lenra-io/server
# fix: Use @me in data transactions

## About this PR

Closes #495 

## Technical highlight/advice

The claims of the transaction token was created with `user` instead of `user_id`.

## How to test my changes

Use this branch in the devtool server for application_runner deps and see the code example in the issue.

## Checklist

### I included unit tests that cover my changes

### I added/updated the documentation about my changes
